edit outdated language and organise community handbook chapters (#4122)
the-turing-way/the-turing-way: book/website/community-handbook/onboarding-calls.md

(ch-onboarding-calls)=

> # Onboarding Calls

Reviewer: I think this call may be in the wrong location! For some reason it shows up for me under the newsletter section?

Author: Not the case for me - see the screenshot of the preview - can you share what you are seeing? 
<img width="333" alt="image" src="https://github.com/user-attachments/assets/e058a0d8-e793-4215-91f1-e81a1367b41c" />

Historically, onboarding in _The Turing Way_ community has happened through 1:1 connections, between the Community Manager and/or Community Leaders and prospective community members of the project.
As the project has grown, so have the number of entry points: where folks may be interacting with different parts of project and community in a variety of different virtual spaces.
Dedicated onboarding calls create an open space for questions, discussions, and a warm welcome into our community.

During this hour-long call, participants learn about _The Turing Way_ project in a group setting and how to contribute to different sub-projects and working groups within the community, and may gain hands-on experience in contributing to _The Turing Way_ guides (in some cases, even making their first pull request!). 

This is a space to bring your questions and interests – we're so excited to meet you!  

### Agenda

> As of January 2024, onboarding calls are being trialed using the following format:

Author: ```suggestion
As of January 2024, onboarding calls use the following format:
```

1. Introduction, icebreaker (15min)
2. The Turing Way - What is it? (5min)
3. How to get involved with The Turing Way? (15min) 
4. Do you have any questions for us? (15min)
5. Feedback (5min)

### Want to onboard other community members? 

Please join the [#onboarding](https://theturingway.slack.com/archives/C06EVT7GKK3) channel on _The Turing Way_ slack! 

We would love to have more community members join in welcoming others to the community.

## Onboarding and Offboarding for Maintenance Level members

> For members joining in Maintenance Level roles, we provide dedicated {ref}`onboarding<ch-onboarding-checklist>` and {ref}`offboarding<ch-offboarding-checklist>` checklists to ensure a smooth transition into and out of their roles.

Reviewer: This is great to add - the cross-citation is really helpful!

These checklists outline key resources and access.

> A meeting with member from The Turing Way using these checklist will facilitate an understanding of expectations upon joining and handover process when departing, helping to maintain the project's continuity and knowledge base.

Author: ```suggestion
A meeting with a member from The Turing Way Maintenance Level using these checklists will facilitate an understanding of expectations upon joining and the handover process when departing, helping to maintain the project's continuity and knowledge base.
```

> A meeting with a member from The Turing Way Maintenance Level using these checklists will facilitate an understanding of expectations upon joining and the handover process when departing, helping to maintain the project's continuity and knowledge base.

Reviewer: ```suggestion
A meeting with a member from _The Turing Way_ Maintenance Level using these checklists will facilitate an understanding of expectations upon joining and the handover process when departing, helping to maintain the project's continuity and knowledge base.
```
